src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1590,19 +1590,19 @@ s32 func_8083224C(PlayState* play) {
     return CHECK_FLAG_ALL(this->actor.flags, ACTOR_FLAG_8);
 }
 
-void func_80832264(PlayState* play, Player* this, LinkAnimationHeader* anim) {
+void Player_AnimPlayOnce(PlayState* play, Player* this, LinkAnimationHeader* anim) {
     LinkAnimation_PlayOnce(play, &this->skelAnime, anim);
 }
 
-void func_80832284(PlayState* play, Player* this, LinkAnimationHeader* anim) {
+void Player_AnimPlayLoop(PlayState* play, Player* this, LinkAnimationHeader* anim) {
     LinkAnimation_PlayLoop(play, &this->skelAnime, anim);
 }
 
-void func_808322A4(PlayState* play, Player* this, LinkAnimationHeader* anim) {
+void Player_AnimPlayLoopAdj(PlayState* play, Player* this, LinkAnimationHeader* anim) {
     LinkAnimation_PlayLoopSetSpeed(play, &this->skelAnime, anim, 2.0f / 3.0f);
 }
```

Comment:
What about using `Slower` instead of `Adj`, so `Player_AnimPlayLoopSlower`? Slightly more descriptive

Reply:
did have this thought as well
I was thinking the name would probably be revisited as it may become clear with docs which anims are formatted this way/if theres any pattern

Do think it may imply that the anim itself will be really slow, when in reality the anims played through these functions look "normal", they were just "encoded" in a way that needs this playback speed

Reply:
"Dilute" ? :sunglasses: :droplet: 
as in, spread the same amount of butter over a larger slice of bread <img src="https://cdn.discordapp.com/emojis/681587643993161762.webp?size=128&quality=lossless" alt="Pepega" height="16"> :fork_and_knife:  :butter:

Reply:
adj is fine and probably accurate to the situation ("adjusted" to look normal despite the weird format)
If we can afford the whole word "Adjusted" it may be better, but I know there are wrapper functions that can cause these to get quite long

Reply:
Adj -> Adjusted seems fine, 4 letters is either okay because it doesn't make the name much longer or the name is already stupid long anyway (hi `Player_AnimChangeOnceMorphAdjZeroRootYawSpeed`)

can also come up with a stupid acronym like ATTS "Adjusted Two Third Speed" or A23S "Adjusted 2/3 Speed" :shrug:

Reply:
Packed, compacted, shrunken, slumped, crushed, squashed, mashed, pressed

Reply:
I think adjusted is fine until we learn more about this. some more docs may reveal some other reason other than "compress them to a smaller size"

Reply:
I've changed Adj -> Adjusted
And added a `PLAYER_ANIM_ADJUSTED_SPEED` define, currently used by Adjusted anim play functions. It looks like it may be used elsewhere in player eventually if the purpose is the same